pew/__main__.py

```diff
@@ -0,0 +1,3 @@
+from pew.pew import pew
+
+pew()
```

Comment:
small aside, the normal python-ism of `if __name__ == '__main__'` doesn't apply here, because the module is *always* named `__main__`

Reply:
i'd still put an if main there

Reply:
Technically you can `from xx import __main__ as not_main` it to change the module name. The check would make a difference in this scenario. Not that you should do that though.